### ISSUE TITLE
KVO doesn't work

### ISSUE THREAD

**Issue by kafejo** [yankodimitrov/SignalKit#17]
I have tried create custom class 

``` swift
class CustomObject: NSObject {
    var age: Int

    init(age: Int) {
        self.age = age
    }
}
```

then create instance

``` swift
self.object = CustomObject(age: 1)
```

then observe the age property

``` swift
self.object.observe().keyPath("age", value: self.object.age).next { print($0) }
```

and finally change age

``` swift
self.object.age += 1
self.object.age += 1
self.object.age += 1
self.object.age += 1
self.object.age += 1
```

But the value is printed only once.

**Comment by yankodimitrov** [comment]
You have to store the chain of operations otherwise it will be disposed.

**Comment by kafejo** [comment]
Could you provide example please? Is it possible to use DisposeBag or I got it wrong? Thanks

**Comment by yankodimitrov** [comment]
Yes you can use <code>DisposableBag</code> to store your observations/bindings here is an example: 

``` swift
class MyClass {

    let disposableBag = DisposableBag()
    // .....

    private func setupObservations() {

        object.observe().keyPath("age", value: object.age)
            .next { print($0) }
            .disposeWith(disposableBag)
    }
}
```

Thanks

**Comment by kafejo** [comment]
Well I tried that too but what I'm getting is `Value of type 'KVOSignal<Int>' has no member 'disposeWith'`

**Comment by kafejo** [comment]
Ah, the 3.0 version hasn't been released on cocoapods, still using 2.0, thats the problem. Thanks for your help! :+1: could you push the new version to cocoapods please?

**Comment by yankodimitrov** [comment]
Just updated the podspec on the cocoapods.

Thanks
